File: Tarea8/abs.js
/***
 * abs.js
 * 
 * return the absolute value of a number
 */

const abs = function(num){
    return num<0 ? num *-1 : num
}

console.log(abs(20)) //20
console.log(abs(0)) //0
console.log(abs(-33)) //33

File: Tarea8/abs.py
#return the absolute value of a number
def abs(num):
    return num *-1 if num<0 else num #operador ternario
    #return number<0 ? numb *-1 : num
    
#Tarea 8: Hacer esto en javaScript
def main():
    print(abs(0))
    print(abs(-3))
    print(abs(10))

if __name__ == "__main__":
    main()

